odl/operator/oputils.py

```diff
@@ -268,6 +269,53 @@ def rmatvec(v):
                                               rmatvec=rmatvec,
                                               dtype=dtype)
 
+
+def as_proximal_lang_operator(op, norm_bound=None):
+    """Wrap ``op`` as a ``proximal.BlackBox``.
+
+    This is intended to be used with the `proximal language solvers.
+    <https://github.com/comp-imaging/ProxImaL>`_
+
+    Parameters
+    ----------
+    op : `Operator`
+        Linear operator to be wrapped. Its domain and range must implement
+        ``shape``, and elements in these need to implement ``asarray``.
+    norm_bound : float, optional
+        An upper bound on the spectral norm of the operator. Note that this is
+        the norm as defined by proximal, and hence use the unweighted spaces.
+
+    Returns
+    -------
+    ``proximal.BlackBox`` : proximal_lang_operator
+        The wrapped operator.
+
+    Notes
+    -----
+    If the data representation of ``op``'s domain and range is of type
+    `NumpyFn` this incurs no significant overhead. If the data type is `CudaFn`
+    or some other nonlocal type, the overhead is significant.
```

Comment:
Would be good to have a link to your nice comparison in the online doc somewhere here. In particular more information on the norm scaling issue mentioned shortly in the `norm_bound` parameter description.